src/settings/useMediaHandler.tsx

```diff
@@ -72,7 +96,9 @@ export function MediaHandlerProvider({ client, children }) {
     );
 
     return {
+      // @ts-ignore, ignore that audioInput is a private members of mediaHandler
       audioInput: mediaHandler.audioInput,
+      // @ts-ignore, ignore that videoInput is a private members of mediaHandler
```

Comment:
thanks for the comment - it's good to mark what's going on with these things so we can later fix either the call or the library, depending on which we think is correct.